### ISSUE TITLE
utox static 64bit for Linux won't start

### ISSUE THREAD

**Issue by jzakiya** [uTox/uTox#875]
The new utox static 0.15 client under PCLinuxOS (PCLOS) with KDE4 desktop won't start.
Below is the error message it gives.
```
[jzakiya@localhost Tox]$ ./utox
./utox: error while loading shared libraries: libfilteraudio.so.0: cannot open shared object file: No such file or directory
[jzakiya@localhost Tox]$ 
```
The previous version worked flawlessly in same system.

However, the same 0.15 client runs correctly in a Virtual Box instance of PCLOS with the KDE5 desktop.

**Comment by robinlinden** [comment]
I fixed this last night. Please try the new version and get back to us if
this problem still exists.

**Comment by jzakiya** [comment]
The new version did correct the startup problem, but when I try to enable video it crashes the app.

**Comment by endoffile78** [comment]
@jzakiya That is known issue and is fixed in #876
